ArcGISRuntimeSDKQt_CppSamples/Geometry/CutGeometry/CutGeometry.qml

```diff
@@ -49,4 +49,20 @@ CutGeometrySample {
             enabled = false;
         }
     }
+
+    Button {
+        id: resetButton
+        anchors {
+            left: parent.left
+            top: parent.top
+            topMargin: 60
+            leftMargin: 10;
+        }
+
+        text: "Reset"
+        onClicked: {
+            resetPolygon();
+            enabled = false;
```

Comment:
This should be disabled by default, enabled when the cut button is clicked, and enable the cut button when this is clicked. 
```suggestion
        enabled: false
        onClicked: {
            resetPolygon();
            enabled = false;
            cutButton.enabled = true;
            // insert resetButton.enabled = true; at line 50 or so
```
Currently the cut button is disabled after it's clicked and the reset button is disabled after it's clicked, leaving the app in a soft-locked state.

![Screen Shot 2022-08-22 at 1 51 09 PM](https://user-images.githubusercontent.com/48941951/186016472-7c196e8a-c1c1-4987-b719-046d6cef31b2.png)